rest-api/src/test/java/com/cloudbees/workflow/rest/endpoints/JobAndRunAPITest.java

```diff
@@ -62,8 +62,8 @@ public void test() throws Exception {
                 "   stage ('Test'); " +
                 "   echo ('Testing'); " +
                 "   stage ('Deploy'); " +
-                ( hudson.Functions.isWindows() ? "   bat ('rmdir /s/q targs || echo no such dir\\n mkdir targs && echo hello> targs\\\\hello.txt'); "
-                                               : "   sh ('rm -rf targs && mkdir targs && echo hello > targs/hello.txt'); "
+                (hudson.Functions.isWindows() ? "   bat ('rmdir /s/q targs || echo no such dir\\n mkdir targs && echo hello> targs\\\\hello.txt'); "
+                        : "   sh ('rm -rf targs && mkdir targs && echo hello > targs/hello.txt'); "
```

Comment:
Noted - this one was inherited, to be honest I'm not sure why there's even a whitespace change here since I don't remember touching this, but... after this many commits, who knows.